src/app/registration/registration.component.ts

```diff
@@ -26,6 +26,8 @@ export class RegistrationComponent implements OnInit {
   isAadharVerified = false;
   aadhaarToken: string;
 
+  myForm: FormGroup;
+
```

Comment:
@apurvaubade Why we need one more formgroup here? can u please remove it, if not being used